ui/apps/platform/src/Containers/Integrations/IntegrationTiles/NotifierIntegrationsSection.tsx

```diff
@@ -31,7 +31,7 @@ function NotifierIntegrationsSection(): ReactElement {
             {descriptors.filter(featureFlagDependencyFilter).map((descriptor) => {
                 const { image, label, type } = descriptor;
                 if (!canUseAcscsEmailIntegration && type === 'acscsEmail') {
-                    return <></>;
+                    return <React.Fragment key={type} />;
```

Comment:
Not a request for change, but would `return null;` be effective here as well?

Reply:
Indeed, super suggestion!